variables.tf

```diff
@@ -16,4 +16,16 @@ variable "global_region" {
 variable "base_dir" {
   type    = string
   default = "./"
-}
\ No newline at end of file
+}
+
+# CloudFront works only with certs stored in us-east-1
+variable "acm_certificate_arn" {
+  type    = string
+  default = ""
+}
+
+variable "cloudfront_aliases" {
+  description = "Your custom domain"
```

Comment:
This is not very precise. Maybe something like "The domain your Next.js page should be available at, e.g. www.mynextjspage.de" would be better?

Reply:
updated